app-backend/db/src/main/scala/com/azavea/rf/database/PlatformDao.scala

```diff
@@ -48,6 +49,34 @@ object PlatformDao extends Dao[Platform] {
       }
     }
 
+  def listPlatformUserTeams(user: User, searchParams: SearchQueryParameters): ConnectionIO[List[Team]] = {
+    val teamsF: Fragment = fr"""
+      id IN (
+        SELECT group_id
+        FROM user_group_roles
+        WHERE
+          group_type = 'TEAM' AND
+          user_id = ${user.id} AND
+          is_active = true
+        )
+    """
+    val organizationsF: Fragment = fr"""
+      organization_id IN (
+        SELECT group_id
+        FROM user_group_roles
+        WHERE
+          group_type = 'ORGANIZATION' AND
+          user_id = ${user.id} AND
+          is_active = true
+        )
+    """
+    TeamDao.query
+      .filter(teamsF ++ fr"OR" ++ organizationsF)
```

Comment:
there's an `or` combinator in `doobie.Fragments` that's a little bit safer here (though this works fine)